eclipse/site/pom.xml

```diff
@@ -171,7 +171,7 @@
       <plugin>
         <groupId>org.jboss.tools.tycho-plugins</groupId>
         <artifactId>repository-utils</artifactId>
-        <version>${jbossTychoPluginsVersion}</version>
+        <version>0.23.1</version>
```

Comment:
sadly there seems to be no repository-utils in that version available. At least my builds fail locally.